src/sidebar/components/top-bar.js

```diff
@@ -20,30 +20,38 @@ const SvgIcon = require('./svg-icon');
 const UserMenu = require('./user-menu');
 
 /**
- * Button for opening/closing the help panel
+ * Reusable component to render a button for toggling a sidebar panel.
+ * Takes an `onClick` callback, as panel toggles are sometimes more complex than
+ * just opening and closing a panel.
  */
-function HelpButton({ onClick }) {
-  const isActive = useStore(
-    store =>
-      store.getState().sidebarPanels.activePanelName === uiConstants.PANEL_HELP
+function TogglePanelButton({ panelName, iconName, title, onClick }) {
+  const currentActivePanel = useStore(
+    store => store.getState().sidebarPanels.activePanelName
   );
+  const isActive = currentActivePanel === panelName;
   return (
     <button
-      className={classnames('top-bar__btn top-bar__help-btn', {
-        'top-bar__btn--active': isActive,
+      className={classnames('top-bar__btn', `top-bar__${iconName}-btn`, {
+        'is-active': isActive,
       })}
       onClick={onClick}
-      title="Help"
-      aria-expanded={isActive}
+      title={title}
       aria-pressed={isActive}
```

Comment:
This only shows up when its `true`
I don't see the state for `aria-presss="false"`. I think its because react is treating the prop as truthy and making it go away when its false.

perhaps `{isActive ? 'true' : 'false'}` will work

Reply:
I find the documentation for `aria-pressed` (in the spec but also in general) to be just a tiny bit vague about what seems to be the fact that it is not a typical Boolean HTML attribute, but a tri-state property (`true/false/undefined`). That can be deduced from the docs, but sheesh :). Anyway, good catch and fixed!